manifests/o/OO-Software/ShutUp10/1.9.1435/OO-Software.ShutUp10.installer.yaml

```diff
@@ -12,9 +11,6 @@ Installers:
   InstallerUrl: https://dl5.oo-software.com/files/ooshutup10/OOSU10.exe
   InstallerSha256: 22D3A45792B749E70B908088E95C19ABAE0707B248FCB83744B23BC6F662425B
   ReleaseDate: 2022-06-02

```

Comment:
```suggestion
```
Remove ReleaseDate as vanity Url is used and wingetbot will try to auto-update the package. In the automation PRs, wingetbot currently doesn't remove ReleaseDate from the manifest, so you would have a newer version with an incorrect ReleaseDate in manifest.